components/molecules/close-contact-warning.tsx

```diff
@@ -1,75 +1,34 @@
 import React, {FC} from 'react';
-import {
-  StyleSheet,
-  TouchableWithoutFeedback,
-  View,
-  Text,
-  Image
-} from 'react-native';
+import {StyleSheet, Text} from 'react-native';
 import {useTranslation} from 'react-i18next';
 import {useNavigation} from '@react-navigation/native';
 
-import {SingleRow} from '../atoms/layout';
-
 import {colors} from '../../constants/colors';
-
-import {shadows, text} from '../../theme';
+import {text} from '../../theme';
+import {Card} from '../atoms/card';
 
 export const CloseContactWarning: FC = () => {
   const {t} = useTranslation();
   const navigation = useNavigation();
 
   return (
-    <TouchableWithoutFeedback
+    <Card
+      padding={{v: 0, h: 0}}
+      type="warning"
+      icon={{
+        w: 107,
+        h: 137,
+        mr: -25,
```

Comment:
I added the new "marginRight" (`mr`) prop to `Card`'s `icon` to set a negative margin and overlap the card's text with the icon. Without it, it would be:

![no_overlap](https://user-images.githubusercontent.com/1233582/89637170-106f7000-d8aa-11ea-9c50-3c14658bffaf.png)

Instead of:

![no vertical padding](https://user-images.githubusercontent.com/1233582/89637190-182f1480-d8aa-11ea-857f-b790d1bc0100.png)